canarytokens/channel_output_email.py

```diff
@@ -323,6 +323,11 @@ def format_report_html(
                 BasicDetails[field_name] = BasicDetails["src_data"].pop(field_name)
         BasicDetails.pop("src_data")
 
+        if "useragent" in BasicDetails and not BasicDetails["useragent"]:
+            BasicDetails.pop("useragent")
+        if "src_ip" in BasicDetails and not BasicDetails["src_ip"]:
+            BasicDetails.pop("src_ip")
+
```

Comment:
are we removing these items from the email? where did we chat that?

Reply:
oh i see only if they're null

Reply:
I discussed this in a [slack thread](https://thinkst.slack.com/archives/C06PKRX50HM/p1721401746099259?thread_ts=1721231935.550169&cid=C06PKRX50HM) with Vittoria. It sounded like it would be easier for her if those items weren't in the data passed to the email to ensure that there won't ever be any blank text boxes in the email.